src/storybook/RouterDecorator.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
+import { StoryFnReactReturnType } from '@storybook/react/dist/client/preview/types'
```

Comment:
don't need to import from internal types. Use `JSX.Element` or `ReactNode` or `ReactElement` or whatever